lib/client.js

```diff
@@ -321,11 +339,7 @@ Client.prototype = {
   /// client.trigger('activation') // activating!
   /// ```
   trigger: function(name, data) {
-    if (this._parent) { throw new Error('Not Implemented'); }
-    if (!this._messageHandlers[name]) { return false; }
-    this._messageHandlers[name].forEach(function(handler) {
-      handler(data);
-    });
+    this.postMessage('iframe.trigger:' + name, { message: data });
```

Comment:
Triggering an event on the same instance works through the framework as well. We could potentially short-circuit it so we manually call the handlers here and ignore incoming events from the framework if they were triggered by the instance, but right now there's no way to tell when events are triggered by the instance.